scripts/image-clean.ps1

```diff
@@ -1,5 +1,8 @@
 Write-Host ('Remove all Docker containers ...')
 docker rm -f $(docker ps -aq);
 
+Write-Host ('Prune all container data ...')
```

Comment:
When do we use this script. It seems very dangerous.

Reply:
There are some tools in `scripts` dir.